repository/user_repository.py

```diff
@@ -1,6 +1,16 @@
-from domain import User, UserType
+from domain import User, UserType, session_scope
 
 
+def check_user_exists(user_id):
```

Comment:
Please put this in a class (don't touch the rest, they should also be in the `UserRepository` class, but we don't want to bother refactoring it all right now)

Reply:
Also please type this